scripts/integrity_check.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
```

Comment:
I don't like the name ”integrity check“. This is a cryptography library and “integrity” has a completely different meaning. If I see an “integrity check” script in a cryptography library, I think that its job is to verify that the code is identical to some point of reference.

Also there are other scripts that fall into the same category of sanity checks, and their naming pattern is `check-thing.extension`.

How about `check-files.py`? It isn't very descriptive, but at least it isn't misleading.